lib/color.zsh

```diff
@@ -0,0 +1,20 @@
+# Define how to colorize before the variables
+prompt_geometry_colorize() {
```

Comment:
`geometry_color_colorize` would follow the format `geometry_<lib>_<function>`. I could also add, as an alias, a `-g-color` function.